src/java/picard/sam/revertsam/RevertSam.java

```diff
@@ -359,56 +379,19 @@ public boolean filterOut(final SAMRecord first, final SAMRecord second) {
         return out;
     }
 
-    /**
-     * Takes an individual SAMRecord and applies the set of changes/reversions to it that
-     * have been requested by program level options.
-     */
-    public void revertSamRecord(final SAMRecord rec) {
-        if (RESTORE_ORIGINAL_QUALITIES) {
-            final byte[] oq = rec.getOriginalBaseQualities();
-            if (oq != null) {
-                rec.setBaseQualities(oq);
-                rec.setOriginalBaseQualities(null);
-            }
-        }
+    private String getDefaultExtension(final String input) {
```

Comment:
I think in this case it makes sense, yes. In general we should be wary of including multiple examples just to show how a particular argument is invoked (eg how we went overboard with SelectVariants in GATK) but these are likely to be equally important use cases. So +1 from me.